src/controllers/userController.js

```diff
@@ -39,6 +48,14 @@ const getSingleUser = async (req, res) => {
 };
 
 const showCurrentUser = async (req, res) => {
+  /*
+     #swagger.summary = 'Returns information about the requesting user based on cookie session
```

Comment:
missing quotation mark at the end is making swagger-autogen parse this weirdly; it's including different stuff than what you have here, but adding the quote mark should fix it (downside of having this syntax in comments is that ide/linter won't catch things like this for us sadly :disappointed: )

```suggestion
     #swagger.summary = 'Returns information about the requesting user based on cookie session'
```

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/f17e7649-a67d-4c11-b03c-f784be9407fc)

Reply:
@akosasante fixed